examples/bevy/README.md

```diff
@@ -0,0 +1,10 @@
+<!-- Copyright © SixtyFPS GmbH <info@slint.dev> ; SPDX-License-Identifier: MIT -->
+
+### `bevy`
+
+This example shows how to integrate Bevy 3D rendering into Slint, using WGPU.
+
+The example can be run on desktop platforms.
+
+![TODO: Screenshot of the Bevy Demo](https://github.com/user-attachments/assets/753c1cc8-ccbf-416f-a130-48b401ff14c0)
+
```

Comment:
A command line to run the example could be nice

The normal way to do `cargo run -p bevy` don't work
```
error: There are multiple `bevy` packages in your project, and the specification `bevy` is ambiguous.
Please re-run this command with one of the following specifications:
  bevy@0.16.0
  bevy@1.12.0
```

But maybe another name should be given:  bevy-example ?

Reply:
Ah yes, I'll rename the example.